setup.cfg

```diff
@@ -7,6 +7,11 @@ addopts = --ignore=scripts
 [flake8]
 ignore = W503, E203, N806
 max-line-length = 100
+per-file-ignores =
+    pyuvsim/tests/*.py: D
+    pyuvsim/*/tests/*.py: D
+    docs/*.py: D,A
+    setup.py: D
 # it's recommended to have max-complexity ~ 18
 max-complexity = 30
-select = B,C,E,W,T4,B9,F
+select = B,C,E,W,T4,B9,F,D
```

Comment:
Instead of "selecting" specific rules, I tend to just *ignore* specific rules (i.e. select everything by default)